### ISSUE TITLE
Tapping off the UIMenu taps the FlatList underneath and causes an onPress event

### ISSUE THREAD

**Issue by lprhodes** [react-native-menu/menu#334]
I had expected that tapping off of the UIMenu would simply dismiss the menu however the FlatList (which renders the UIMenu in each row) processes the tap event and causes an action at the same time as dismissing the UIMenu.

How are people handling this?

**Comment by MaxInMoon** [comment]
@lprhodes any news on this issue? 👀

**Comment by edenmoyal1** [comment]
@lprhodes found a solution?

**Comment by lprhodes** [comment]
No, sorry - I’ve just accepted that it’s working that way for now.
